robot/rospackages/src/mcu_control/scripts/NavigationNode.py

```diff
@@ -19,56 +19,69 @@ def subscriber_callback(message):
         rover['heading'] = message.z
         hasHeading = True
 
-    if gotGpsPos:
-        Rov_to_des_distance = Distance(rover['latitude'], rover['longitude'], \
-        gpsGoal['latitude'], gpsGoal['longitude'])
-        Rov_to_des_direction = Direction(rover['latitude'], rover['longitude'], \
-        gpsGoal['latitude'], gpsGoal['longitude'])
+    if rospy.get_param('has_gps_goal'):
+        rov_to_des_distance = Distance(rover['latitude'], rover['longitude'],
+                                       gpsGoal['latitude'], gpsGoal['longitude'])
+        rov_to_des_direction = Direction(rover['latitude'], rover['longitude'],
+                                         gpsGoal['latitude'], gpsGoal['longitude'])
 
         if hasHeading:
```

Comment:
no :camel: case for python variables, :snake: case only :eye:

Reply:
Fixed